deno/rest/v10/channel.ts

```diff
@@ -584,8 +584,12 @@ export type RESTPostAPIChannelMessagesThreadsJSONBody = AddUndefinedToPossiblyUn
 /**
  * https://discord.com/developers/docs/resources/channel#start-thread-in-forum-channel
  */
-export type RESTPostAPIGuildForumThreadsJSONBody = RESTPostAPIChannelMessagesThreadsJSONBody &
-	RESTPostAPIChannelMessageJSONBody;
+export type RESTPostAPIGuildForumThreadsJSONBody = RESTPostAPIChannelMessagesThreadsJSONBody & {
+	/**
+	 * First message in the forum thread
+	 */
+	message: RESTPostAPIChannelMessageFormDataBody;
```

Comment:
This needs to be JSONBody not formdata body

Reply:
Oops, I misread